### ISSUE TITLE
Theme Browser: space between search bar and theme list

### ISSUE THREAD

**Issue by ScoutHarris** [wordpress-mobile/WordPress-iOS#9730]
### Expected behavior
When searching, the top of the themes list should be right under the search bar.

### Actual behavior
When searching, there is a gap between the top of the themes list and the search bar. This is most severe on the iPhone. There is a small gap on the iPad, just not as bad.

### Steps to reproduce the behavior
- Go to Site > Themes.
- Select the Search bar.
- Notice a gap between the search bar and the first theme.
- Drag the view down as if you're scrolling to the top. 
- Notice the gap gets even bigger. 

Initial gap:
![initial](https://user-images.githubusercontent.com/1816888/42482634-f1061c6a-83a5-11e8-8970-50e3b4e6850e.png)

Gap after scrolling:
![after_scrolling](https://user-images.githubusercontent.com/1816888/42482640-fb79c840-83a5-11e8-9880-c0f8df238313.png)

**Comment by ScoutHarris** [comment]
In `ThemeBrowserViewController` there is a workaround for this gap to show the `noResultsViewController` correctly:

```
// There is a gap between the search bar and the collection view - https://github.com/wordpress-mobile/WordPress-iOS/issues/9730
// This makes the No Results View look vertically off-center. Until that is resolved, we'll move the NRV up by the search bar height.
noResultsViewController.view.frame.origin.y -= searchBarHeight
```

This should be removed when this issue is fixed.

**Comment by kean** [comment]
Fixed in 25.3.
